src/components/resources.js

```diff
@@ -165,7 +165,7 @@ function Resources(props) {
             });
           });
         });
-        return array.map((x, i) => (
+        return array.sort().map((x, i) => (
```

Comment:
Seems looks to fine. But I checked in entire code of this file. And find we are using multiple foreach inside. That is running three loop inside. Which is total incorrect. As we know we are going to large level data. So we should consider small things that could be impact at performance of our application. 
See below example -

![image](https://user-images.githubusercontent.com/29035159/79717647-e0e72880-82f7-11ea-8abe-3c9f3f37c597.png)

We could use loadash function to handle this scenario. That provide linq operation.

@samyakmehta @nikita-11